src/components/CopyrightInfo/CopyrightInfo.module.scss

```diff
@@ -0,0 +1,47 @@
+@import 'global.scss';
```

Comment:
@jambao24 Just as a note, we shouldn't blindly copy and paste SCSS. For Copyrightinfo, we only need the `Copyright` css class, and nothing else.